synapse/models/crypto.py

```diff
@@ -13,7 +13,26 @@
 class CryptoModule(s_module.CoreModule):
     def getModelDefs(self):
         modl = {
+
```

Comment:
To discuss: an "account" or "wallet". There's can be a many-to-one relationship between the wallet and public keys it uses for sending.

Reply:
I specifically chose "address" because in most currencies it is the thing with a 1-to-1 with a given public key.  These address nodes are not meant to represent a wallet, but a specific address as it appears within a transaction.